website/docs/en/plugins/rspack/copy-rspack-plugin.mdx

```diff
@@ -2,7 +2,7 @@ import { Table } from '@builtIns';
 import WebpackLicense from '@components/webpack-license';
 import { ApiMeta } from '@components/ApiMeta.tsx';
 
-<WebpackLicense from="https://webpack.js.org/plugins/copy-webpack-plugin/#root" />
+{/* <WebpackLicense from="https://webpack.js.org/plugins/copy-webpack-plugin/#root" /> */}
```

Comment:
Is this a mistake?

Reply:
Source page: https://rspack.dev/plugins/rspack/copy-rspack-plugin

this plugin is `rspack` only, so it seems unnecessary to show a license block